server/tests/delta_test.rs

```diff
@@ -0,0 +1,159 @@
+mod delta_test {
+    use actix_http::header::IF_NONE_MATCH;
+    use actix_http::HttpService;
+    use actix_http_test::{test_server, TestServer};
+    use actix_service::map_config;
+    use actix_web::dev::AppConfig;
+    use actix_web::http::header::{ETag, EntityTag};
+    use actix_web::{web, App, HttpRequest, HttpResponse};
+    use chrono::Duration;
+    use dashmap::DashMap;
+    use std::sync::Arc;
+    use unleash_edge::feature_cache::FeatureCache;
+    use unleash_edge::http::feature_refresher::FeatureRefresher;
+    use unleash_edge::http::unleash_client::{ClientMetaInformation, UnleashClient};
+    use unleash_edge::types::EdgeToken;
+    use unleash_types::client_features::{
+        ClientFeature, ClientFeatures, ClientFeaturesDelta, Constraint, Operator, Segment,
+    };
+    use unleash_yggdrasil::EngineState;
+
+    #[actix_web::test]
+    #[tracing_test::traced_test]
+    async fn test_delta() {
```

Comment:
Testing whether, after the eTag is provided by Edge, Edge can successfully apply the delta when Unleash sends a new delta.